app/code/Magento/Cms/Test/Mftf/ActionGroup/VerifyCmsPageSaveSplitButtonActionGroup.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="VerifyCmsPageSaveSplitButtonActionGroup">
```

Comment:
Please, rename this action group to match the follwing name:

```
AssertAdminCmsPageSaveSplitButtonActionGroup
```

Thank you.

Reply:
@rogyar I have resolved this as per your comment.